portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/CreateABTest.testcase

```diff
@@ -55,66 +46,53 @@ definition {
 	@description = "Validate if the user can create an A/B Test"
 	@priority = "5"
 	test CreateABTestByBounceRate {
+		ABTest.createABTestByBounceRate(testTitle = "AB test title");
```

Comment:
change this testTitle to "Create A/B Test Title"
according to the line 63.
		AssertTextEquals(
			locator1 = "ABTest#AB_TEST_TITLE_AC_SIDE",
			value1 = "Create A/B Test Title");